apps/portal/prisma/new-schema.prisma

```diff
@@ -0,0 +1,186 @@
+}
+
+model User {
+    id            String        @id @default(auto()) @map("_id") @db.ObjectId
+    name          String?
+    email         String?       @unique
+    emailVerified DateTime?
+    image         String?
+    accounts      Account[]
+    sessions      Session[]
+    profile       Profile?
+    roles         RolesOnUser[]
+
+    user_app UserAppJunction @relation(fields: [userId], references: [id])
+    @@map("users")
+}
+
+model VerificationToken {
+    id         String   @id @default(auto()) @map("_id") @db.ObjectId
+    identifier String
+    token      String   @unique
+    expires    DateTime
+
+    @@unique([identifier, token])
+}
+
+model Profile {
+    id               String    @id @default(auto()) @map("_id") @db.ObjectId
+    firstName        String
+    lastName         String
+    email            String    @unique
+    netid            String    @unique
+    classStanding    String
+    major            String
+    utdStudent       Boolean
+    roles            String[]
+    membershipStatus Boolean
+    membershipTS     DateTime?
+    resume           Boolean
+    resumeTS         DateTime?
+
+    user User @relation(fields: [userId], references: [id])
+    officer Officer @relation(fields: [profileId], references: [id])
+
+    userId String             @unique @db.ObjectId
+    events EventReservation[]
+}
+
+model Event {
+    id          String             @id @default(auto()) @map("_id") @db.ObjectId
+    summary     String
+    description String
+    url         String
+    location    String
+    start       DateTime
+    end         DateTime
+    profiles    EventReservation[]
+
+    @@map("events")
+}
+
+model TypeformApplication {
+    id                  String  @id @default(auto()) @map("_id") @db.ObjectId
+    active              Boolean
+    description         String
+    endpoint            String
+    externalResourceUrl String
+    typeformId          String
+    typeformName        String
+
+    user_app UserAppJunction @relation(fields: [formmId], references: [typeformId])
+    @@map("typeform_applications")
+}
+
+model UserAppJunction {
+    id                  String  @id @default(auto()) @map("_id") @db.ObjectId
+    entryId             String
+    userId              String
+    formId              String
+    deptId              String
+    interId             String
+    responses           String
+    status              String
+    score               Int
+    first               String
+    second              String
+    third               String
+    notes               String
+}
+
+model Interview {
+    id                  String  @id @default(auto()) @map("_id") @db.ObjectId
+    interDate           String
+    status              String
+    link                String
+}
+
+
+model Department {
+    id                  String  @id @default(auto()) @map("_id") @db.ObjectId
+    deptName            String
+
+    officer Officer @relation(fields: [deptId], references: [id])
```

Comment:
Based on schema, `Department` and `Officer` have many-to-many relationship, meaning that this should go to an intermediary table instead of the `Officer` model directly.

Reply:
you are right - i mistakingly saw that as one to many. fixing it!